modules/administration/pages/kubernetes.adoc

```diff
@@ -75,21 +72,20 @@ If the node data is not available wait a few moments and try again.
 
 See the following steps to find runtime data for images.
 
-. Select menu:Images[Images] from the navigation menu.
-. In the image list table, take notice of the new runtime columns.
-These are labeled: ``Revision``, `Runtime` and ``Instances``.
-Initially these columns will not provide useful data.
+. Select menu:Images[Image List] from the navigation menu.
+. In the image list table, the runtime columns are labeled ``Revision``, ``Runtime``, and ``Instances``.
+Initially, these columns do not provide useful data.
 
-** ``Revision``: An artificial sequence number which increments on every rebuild for manager-built images, or on every reimport for externally built images.
+** ``Revision``: An artificial sequence number that increments on every rebuild for manager-built images or on every re-import for externally built images.
 ** ``Runtime``: Overall status of the running instances of the image throughout the registered clusters.
 The status can be one of the following:
-*** All instances are consistent with SUSE Manager: All the running instances are running the same build of the image as tracked by SUSE Manager.
-*** Outdated instances found: Some of the instances are running an older build of the image.
+*** _All instances are consistent with SUSE Manager_: All the running instances are running the same build of the image as tracked by {productname}.
+*** _Outdated instances found_: Some of the instances are running an older build of the image.
 A redeploy of the image into the pod may be required.
-*** No information: The checksum of the instance image does not match the image data contained in SUSE Manager.
+*** _No information_: The checksum of the instance image does not match the image data contained in {productname}.
 A redeploy of the image into the pod may be required.
-** ``Instances``: Number of instances running this image across all the clusters registered in SUSE Manager.
-A breakdown of numbers can be seen by clicking on the pop-up icon next to the number.
+** ``Instances``: Number of instances running this image across all the clusters registered in {productname}.
+A breakdown of numbers can be seen by clicking the pop-up icon next to the number.
```

Comment:
I don't understand the nesting here. If you're nesting three deep in a procedure, the procedure needs a re-write.

Reply:
Most of this probably belongs into the Reference guide.

Reply:
Possibly. I think we should probably create an issue and follow up on this one. It's going to need more work than we can really do in this review.

Reply:
ok, committed a minor simplification.  Ready to be pushed.